drgn/helpers/linux/kernel_config.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+"""
+Kconfig
+------
+
+
+The ``drgn.helpers.linux.kconfig`` module provides helpers for reading kernel
```

Comment:
This says the module is `kconfig` but the file is called `kernel_config.py`. Let's stick with `kconfig.py`.